### PR TITLE
[Experiment] Use Jbang camel k plugin in e2e test

### PR DESCRIPTION
**Pull request by gansheer**
**Release Note**
```release-note
NONE
```

### CODE REVIEW

**Review by squakez — COMMENTED** [review]
LGTM, nice work! With this approach we can keep `kamel` test as well while we keep supporting other CLI and be able to start using `camel` in all the `run` tests.

File: .github/actions/setup-camel-jbang/install-jbang.sh [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+####
+#
+# Outputs the kind config to output variables
+#
+####
+
+set -e
+
+curl -Ls https://sh.jbang.dev | bash -s - app setup
```
[the remark this answers is not in the document]
Comment (gansheer) [reply .github/actions/setup-camel-jbang/install-jbang.sh]: That was a leftover from early work

**Review by claudio4j — APPROVED** [review]

**Comment by gansheer** [comment]
@squakez @oscerd @christophd @claudio4j  Could the tests be triggered please :pray:

**Comment by squakez** [comment]
@gansheer I understand this can be closed given that the `camel k` plugin has been deprecated, is that correct?

**Comment by gansheer** [comment]
> @gansheer I understand this can be closed given that the `camel k` plugin has been deprecated, is that correct?

Yes, let's close this.

**Pull request closed without merging**